docs/DeveloperGuide.md

```diff
@@ -283,32 +283,63 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `ThunderCat AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case1: UC01 - Input the detail of group mate**
 
 **MSS**
+1. User type adding command into the terminal
+2. ThunderCat announces that the contact is successfully added
+   Use case ends.
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+**Extensions**
+* 1a. ThunderCat detects an error in the entered command.
+  * 1a1. ThunderCat requests for the correct command, and gives examples.
+  * 1a2. User enters a new command.
+    Steps 1a1-1a2 are repeated until the commands entered are correct.
+    Use case resumes from step 2.
+  
+**Use case: UC02 - Totally delete the contact**
 
-    Use case ends.
+**MSS**
+1. User enters the user's ID to be deleted.
+2. ThunderCat makes warning about the contact that will be deleted, and all groups that the contact is in.
+3. User confirms the deletion.
+4. ThunderCat announces that the address is successfully deleted.
+   Use case ends.
 
 **Extensions**
+* 1a. ThunderCat detects an error in the entered command (UC01 - extension 1a).
```

Comment:
I'm not completely sure if you can include part of a use case (like in this case, an extension of a use case, etc.) We should probably clarify that, but it seems legit to me so maybe I can just accept it for now.